app/views/staff/tracks/_form.html.haml

```diff
@@ -1,10 +1,8 @@
-= simple_form_for [ event, :staff, track ], remote: true do |f|
-  .modal-body
-    = f.input :name, placeholder: 'Name'
-    = f.input :description, as: :text, input_html: {rows: 6}, placeholder: 'Description',
-      hint: 'Brief description (fewer than 250 characters)'
-    = f.input :guidelines, as: :text, input_html: {rows: 12}, placeholder: 'Guidelines',
-      hint: 'A more detailed description of the track that will appear in the Event Guidelines.'
-  .modal-footer
-    %button.pull-right.btn.btn-primary{:type => "submit"} Save
-    %button.btn.btn-default{'data-dismiss' => "modal"} Cancel
\ No newline at end of file
+= f.input :name, placeholder: 'Name'
```

Comment:
Since the form itself isn't rendered in the partial anymore, you might want to rename it to _fields